docs/examples/guide/input/mouse01.py

```diff
@@ -22,7 +12,11 @@ class MouseApp(App):
 
     def compose(self) -> ComposeResult:
         yield RichLog()
-        yield PlayArea(Ball("Textual"))
+        yield Ball("Textual")
+
+    def on_mouse_move(self, event: events.MouseMove) -> None:
+        self.screen.query_one(RichLog).write(event)
+        self.query_one(Ball).offset = event.screen_offset - (8, 2)
```

Comment:
Using `event.offset` will make the "ball" alternate between the top-left corner of the screen and the mouse position and the mouse event happens inside the "ball".